spec/features/avo/has_many_field_spec.rb

```diff
@@ -154,4 +154,29 @@
       }.not_to raise_error
     end
   end
+
+  describe "through relationship" do
+    let!(:team) { create :team }
+    let!(:user) { create :user }
+
+    it "triggers callbacks on through model" do
+      team.team_members << user
+      expect(team.team_members.count).to eq 1
+      team_membership = team.memberships.first
+
+      visit "/admin/resources/teams/#{team.id}/team_members?view=show&turbo_frame=has_many_field_show_team_members"
+
+      expect { find("tr[data-resource-id='#{user.to_param}'] [data-control='detach']").click }.to raise_error("Callback Called")
+    end
+
+    it "triggers callbacks when called from the other model too" do
```

Comment:
Yes it works both ways of the association @Paul-Bob. This test verifies that :sweat_smile: